OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -979,8 +900,8 @@ else if (pushStatusRuntimeError(subscribableStatus))
       });
    }
 
-   private static boolean pushStatusRuntimeError(int subscribableStatus) {
-      return subscribableStatus < -6;
+   private static boolean pushStatusRuntimeError(int subscriptionStatus) {
```

Comment:
`subscribableStatus` was an intended name here. It isn't if the device is subscribed but if it could be.